lib/private/Files/Node/Node.php

```diff
@@ -297,10 +297,18 @@ public function getParent(): INode|IRootFolder {
 				return $this->root;
 			}
 
+			// Manually the parent if the current node doesn't have a file info yet
```

Comment:
```suggestion
			// Manually fetch the parent if the current node doesn't have a file info yet
```
Or get? But a word is missing there I think.

Reply:
Yep, fixed